src/program/config/purge_alarms/README

```diff
@@ -0,0 +1,32 @@
+Usage: snabb config purge-alarms [OPTION]... ID STATE
+Adds a new operator-state in an alarm.
+
+Available options:
+  -s, --schema SCHEMA           YANG data interface to request.
+  -r, --revision REVISION       Require a specific revision of the YANG module.
+  -f, --format                  Selects output format (yang or xpath). Default: yang.
+      --print-default           Forces print out of default values.
+  -h, --help                    Displays this message.
+
+Filtering options:
+  --by-severity                 [severity-spec/value].
```

Comment:
OK this is a nit-pick but what do you think about using a colon (`:`) instead of a slash to separate the spec and the value?  Here and below.  Seems to fit better with other "search" text interfaces.

Reply:
SGTM. There's one filtering operation though (operator-state-filter) that admits two different types of values (state and user). I used a slash to separate them in case an user wants to use both values. I'm not sure about this.